pkg/git/git.go

```diff
@@ -309,7 +309,7 @@ func (g *Git) fetchAndReset(rev string) error {
 }
```

Comment:
Change is fine, confused on branching though. If v1.0.2 was tagged off of branch release-v1.0, then what is master branch ? Maybe a head ? We could use more docs :)

Reply:
Matt has this issue for branching up https://github.com/rancher/wrangler/issues/250